src/caduceus/http.go

```diff
@@ -35,16 +37,16 @@ func (sh *ServerHandler) ServeHTTP(response http.ResponseWriter, request *http.R
 		response.WriteHeader(http.StatusBadRequest)
 		response.Write([]byte(fmt.Sprintf("Unsupported method \"%s\"... Caduceus only supports \"POST\" method.\n", request.Method)))
 
-		debugLog.Log(messageKey, "Unsupported method", "method", request.Method, "explanation",
-		"Caduceus only supports \"POST\" method.")
+		errorLog.Log(errorKey, "Unsupported method", "method", request.Method, messageKey,
```

Comment:
@johnabass This log did not seem a debug level (as it was before switching to go-kit) one but rather that of an error. Correct if I am wrong.

Reply:
This log should be removed entirely.  The mapping for the handler should prevent unsupported methods up front, so code should never get into this case.

Reply:
Addressed.